hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -62,6 +62,7 @@
     public static final String PROP_VERSION_CHECK_ENABLED = "hazelcast.version.check.enabled";
     public static final String PROP_PREFER_IPv4_STACK = "hazelcast.prefer.ipv4.stack";
     public static final String PROP_IO_THREAD_COUNT = "hazelcast.io.thread.count";
+    public static final String PROP_IO_HANDLER_MIGRATION_INTERVAL_SECONDS = "hazelcast.io.handler.migration.interval.seconds";
```

Comment:
Can you add more documentation to the property? For external people this property will be magic. And would it not make sense to call it e.g. PROP_IO_IMBALANCE_DETECTOR_INTERVAL_SECONDS. That there is an IO handler is an implementation detail. Also that a migration happens is an implementation detail. 

With this name you focus on what is happening, instead of how it is happening. 

Perhaps there are better names than the one I came up with.

We also need the ability to disable the mechanism when for whatever reason it causes problems.